modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/GridExchangeFreeSwitchTest.java

```diff
@@ -197,7 +216,11 @@ private void testNodeLeftOnFullyRebalancedCluster() throws Exception {
 
             awaitPartitionMapExchange(true, true, null, true);
 
-            assertEquals(persistence ? 0 /*PME absent*/ : (nodes - 1) /*regular PME*/, cnt.get());
+            boolean absentPme = persistence && !pmeFreeDisabled;
```

Comment:
Duplicated absentPme calculation.
Lets calculate this once.

Reply:
done